### PR TITLE
[doc] Fix the link to block disable in useless-import-alias related

### PR DESCRIPTION
**Pull request by Pierre-Sassoulas**
## Type of Changes

|     | Type                   |
| --- | ---------------------- |
| ✓   | :bug: Bug fix          |

## Description

Apparently the redirect in #6628 is not perfect.

### CODE REVIEW

**Review by DanielNoord — APPROVED** [review]
First time `linkcheck` helped us 😄

**Comment by Pierre-Sassoulas** [comment]
Link that need to work before we merge (will update):
https://pylint--6642.org.readthedocs.build/en/6642/messages/messages_list.html 
https://pylint--6642.org.readthedocs.build/en/6642/messages/messages_introduction.html

File: doc/data/messages/u/useless-import-alias/related.rst [hunk 1]
```diff
@@ -1,3 +1,3 @@
 - `PEP 8, Import Guideline <https://peps.python.org/pep-0008/#imports>`_
-- `Pylint block-disable <https://pylint.pycqa.org/en/latest/user_guide/message-control.html#block-disables>`_
+- `Pylint block-disable <block_disables>`_
```
Comment (DanielNoord) [inline doc/data/messages/u/useless-import-alias/related.rst:2]:
```suggestion
- :ref:`Pylint block-disable <messages-control:Block disables>`_
```

File: doc/data/messages/u/useless-import-alias/related.rst [hunk 2]
```diff
@@ -1,3 +1,3 @@
 - `PEP 8, Import Guideline <https://peps.python.org/pep-0008/#imports>`_
-- `Pylint block-disable <https://pylint.pycqa.org/en/latest/user_guide/message-control.html#block-disables>`_
+- :ref:`Pylint block-disable <messages-control:Block disables>`_
```
Comment (DanielNoord) [inline doc/data/messages/u/useless-import-alias/related.rst:2]:
```suggestion
- :ref:`Pylint block-disable <block_disables>`_
```

File: doc/data/messages/u/useless-import-alias/related.rst [hunk 3]
```diff
@@ -1,3 +1,3 @@
 - `PEP 8, Import Guideline <https://peps.python.org/pep-0008/#imports>`_
-- `Pylint block-disable <https://pylint.pycqa.org/en/latest/user_guide/message-control.html#block-disables>`_
+- :ref:`Pylint block-disable <block_disables>`_
```
Comment (DanielNoord) [inline doc/data/messages/u/useless-import-alias/related.rst:2]:
```suggestion
- :ref:`Pylint block-disable <block_disables>`
```

**Comment by Pierre-Sassoulas** [comment]
Can we revert #6647, if sphinx-reredirect works with relative path ? :)

**Comment by DanielNoord** [comment]
> Can we revert #6647, if sphinx-reredirect works with relative path ? :)

😅 

We might. Let me retry!

**Comment by Pierre-Sassoulas** [comment]
Sorry I pushed something that cut the pipeline, I'll let you work on this issue, let me know if I can help. I'm testing with https://github.com/PyCQA/pylint/pull/6642#issuecomment-1131417982 right now, but we might need to add real CI tests later.

**Pull request merged**